examples/notebook/deploy-to-gcp.ipynb

```diff
@@ -237,4 +240,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 2
-}
+}
```

Comment:
nit: Add new line. Same below.. Thanks

Reply:
Thanks, updated, seems it was caused by my vscode auto deleted it.